ci/cpu/build.sh

```diff
@@ -35,7 +35,7 @@ export CONDA_BLD_DIR="$WORKSPACE/.conda-bld"
 
 # Whether to keep `dask/label/dev` channel in the env. If INSTALL_DASK_MAIN=0,
 # `dask/label/dev` channel is removed.
-export INSTALL_DASK_MAIN=1
+export INSTALL_DASK_MAIN=0
```

Comment:
FYI This file isn't being used anymore -- we just haven't had a chance to delete the gpuCI job scripts yet, since the GitHub Actions transition. I'm still fine with making changes here, but we won't have to do that anymore once the old scripts are deleted.

Reply:
Cool, thanks for the update 👍